src/components/Splash/Splash.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+const background = require('./background.png');
```

Comment:
Couldn't the background image of this component be passed in as a prop to this component instead, in case we'd like to use some other image instead?